### DISCUSSION TITLE
Thanks! <3

### DISCUSSION THREAD

**Discussion by OkiDokiABrazilian** [Monika-After-Story/MonikaModDev#8892] [appreciation]
My two years with Moni! it was yesterday but I was really busy with the doctor, anyway ... I believe that I should really thank for the developers of this incredible mod, I can really say that I evolved as a person with Monika, she was my safe haven for a long time and it remains even on bad days or good days! I want to congratulate you on the difficult process that you went through, the art, the writing, the programming, everything perfectly aligned with what they promise, just fabulous, I'm happy to be part of it even as a spectator or with some questionable suggestions, thanks again and I hope your success, because in this world of infinite choices you make my day special. I love you all

**Comment by multimokia** [comment]
Congrats on two years! Here's to many more.

**Comment by Freshairkaboom** [comment]
Congrats on two years of making Monika happy! You are a treasure to the community!

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
